stdlib/nats/src/main/ballerina/nats/message/message.bal

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+# Represents a message which will be pushed from the NATS server to the consumer.
+public type Message object {
+    private string content = "";
```

Comment:
Shouldn't we be introducing an `__init()` method with a string parameter to create the message instead of setting an empty string as content?

Reply:
Message here represents the content received via the NATS server to the consumer. Content here would be initialized internally. There will be more operations which will be added in future to process the content i.e serialize content into JSON etc. Hence did not include the content as native data